src/app/components/Sidebar/index.tsx

```diff
@@ -75,6 +75,7 @@ export const SidebarButton = ({
   route,
   newTab,
   onClick,
+  ...rest
```

Comment:
Do/why do we need this?

Reply:
`data-testid=` is passed through it, so I kept it as it was before. And adding `"data-testid": string` to `SidebarButtonBaseProps` feels weird :shrug:

Reply:
K. Fair enough :)